ReactWindows/ReactNative/Views/TextInput/ReactTextInputManager.cs

```diff
@@ -212,6 +212,19 @@ public void SetPlaceholder(ReactTextBox view, string placeholder)
         }
 
         /// <summary>
+        /// Sets the background color for the <see cref="ReactTextBox"/>.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="color">The masked color value.</param>
+        [ReactProp(ViewProps.BackgroundColor, CustomType = "Color")]
+        public void SetBackgroundColor(ReactTextBox view, uint? color)
+        {
+            view.Background = color.HasValue
+                ? new SolidColorBrush(ColorHelpers.Parse(color.Value))
+                : null;
```

Comment:
Is null the right behavior here?  Shouldn't the background color of a TextBox be white by default?